### PR TITLE
Completed first draft of slides 

### PR DESCRIPTION
**Pull request by matteociccozzi**
added slides on eigenvalue placement, ccf, and linearization of vector ODE systems

### CODE REVIEW

File: mt2.tex [hunk 1]
```diff
@@ -18,4 +22,373 @@
...
+	\end{frame}
+
+	\begin{frame}{Steps to Linearize Vector ODE Systems}
+	To linearize $\frac{d\vec{x}}{dt} = \vec{f}(\vec{x}(t), \vec{u}(t))$ we use a similar procedure as we did for the scalar case. \pause 
+	\begin{enumerate}[(i)]
+	    \item If you're not given a DC input $\vec{u}^*$, determine one. \pause \\
+	    \item Solve $\vec{f}(\vec{x}^*, \vec{u}^*) = \vec{0}$ to determine the DC operating point. \pause \\
+	    \item Let $\vec{x}_l(t) = \vec{x}(t) - \vec{x}^*$ and $\vec{u}_l(t) = \vec{u}(t) - \vec{u}^*$ \pause \\
+	    \item Linearize $\vec{f}(\vec{x}, \vec{u})$ about $(\vec{x}^*, \vec{u}^*)$. That is: $\vec{f}(\vec{x}(t), \vec{u}(t)) \approx \vec{f}(\vec{x}^*, \vec{u}^*) + J_{\vec{x}}\vec{x}_l(t) + J_{\vec{u}}\vec{u}_l(t)$ \pause \\
+	    \item Plug (iv) back into the ODE: $\frac{d\vec{x}}{dt} \approx \cNCEL{\vec{f}(\vec{x}^*, \vec{u}^*)} + J_{\vec{x}}\vec{x}_l(t) + J_{\vec{u}}\vec{u}_l(t)$
```
Comment (GeoHutch) [inline mt2.tex]: What is cNCEL ? It's not compiling on my environment

Comment (matteociccozzi) [reply mt2.tex]: should be \cancel sorry!

File: mt2.tex [hunk 2]
```diff
@@ -18,4 +22,373 @@
...
+	\begin{frame}
+		\frametitle{Eigenvalue Placement} 
+  		\centering\includegraphics[width = 10cm]{eigenvalue.jpg}
+	\end{frame}
+
+	\begin{frame}
+	\frametitle{Why?} 
+		\begin{itemize}
+		    \item Recall that we are always interested in determining if a given system is BIBO (bounded input bounded output) stable. 
+		    \item More precisely, if we have a system described by $\vec{x}(t+1) = A\vec{x}(t) + Bu(t) + \vec{\omega}(t)$ we would like the eigenvalues of $A \in \R^{nxn}$, to satisfy the following property : $|\lambda_i| < 1$.
```
Comment (PTNobel) [inline mt2.tex:35]: Please replace \R^{nxn} with \R^{n\times n}

File: mt2.tex [hunk 3]
```diff
@@ -18,4 +22,373 @@
...
+		    \item This is where eigenvalue placement comes into play!
+		    \item Assuming the system is controllable, we will use closed loop controls to change the eigenvalues such that they satisfy this property. 
+		\end{itemize}
+	\end{frame}
+
+	\begin{frame}
+	\frametitle{How?}
+	\begin{itemize}
+	    \item $u(t)$ is the input in the system. If the system is controllable then we can use feedback, which means that we can let the input depend on the output, $\vec{x}(t)$.
+	    \item We would like to change the matrix multiplying $\vec{x}(t)$ such that  $|\lambda_i|<1$, so let's see what happens when we let $u(t) = K\vec{x}(t)$, where $K \in \R^{1xn}$.
```
Comment (PTNobel) [inline mt2.tex:46]: 1xn should be 1 \times n

File: mt2.tex [hunk 4]
```diff
@@ -18,4 +22,373 @@
...
+	    $\frac{dx_l(t)}{dt} = -x_l(t) + u_l(t)$ 
+	\end{enumerate}
+	 \pause 
+	 We see that our assumption that  $x_l(t)$ is small is indeed satisfied as we will have a $e^{-t}$ term in the solution which means that $x_l(t)$ will decay.\\ \pause
+	What if we had chosen a different DC Operating point, say $-\frac{\pi}{2}$? When we linearize the system we see that the solution will "explode" around that particular DC operating point.
+	\end{frame}
+
+	\begin{frame}{Linearization of Vector Functions}
+	What if we had $\frac{d\vec{x}}{dt} = \vec{f}(\vec{x}, \vec{u})$ ? We will see that the process is very similar to the scalar case we just analyzed!\\ \pause
+	First, let's see how to linearize $\vec{f}(\vec{x})$ around a DC operating point $\vec{x}^*$. Where $\vec{f} \in \R^{nx1}$ is a vector of scalar functions. \\\pause
```
Comment (PTNobel) [inline mt2.tex:254]: nx1 -> n \times 1

File: mt2.tex [hunk 5]
```diff
@@ -18,4 +22,373 @@
...
+	\\
+
+	\end{frame}
+
+	\begin{frame}{Linearization with Jacobians Example}
+	    \begin{align*}
+	    \text{Linearize}  \hspace{0.2 cm} 
+	     \vec{f}(\vec{x}(t))  = 
+	    \begin{bmatrix}
+	    sin(x_1(t)*x_2(t)) + 2x_1(t)x_3^2(t)\\
```
Comment (PTNobel) [inline mt2.tex:279]: \sin and \cos instead of sin and cos in this line and the one below

File: mt2.tex [hunk 6]
```diff
@@ -18,4 +22,373 @@
...
+
+	\begin{frame}{Solutions}
+	 Find the Jacobian: 
+	\small
+	\[
+	        \begin{bmatrix}
+	        x_2(t)cos(x_1(t)*x_2(t)) + 2x_3^2(t)& x_1(t)cos(x_1(t)*x_2(t)) & 4x_1(t)x_3(t)\\ 
+	        \frac{1}{x_3(t)}& -x_3(t)sin(x_2(t)) &cos(x_2(t)) - \frac{x_1(t)}{x_3^2(t)}  \\ 
+	        1 & 6x_3(t)x_2^2(t)& 2 x_2^3(t) 
+	        \end{bmatrix}
```
Comment (PTNobel) [inline mt2.tex:297]: Should be \sin and \cos for all of these

File: mt2.tex (hunk 5 above)
Comment (PTNobel) [reply mt2.tex:279]: Also, this could just be me but it feels like a lot more algebra then the class normally requires.

File: mt2.tex (hunk 6 above)
Comment (PTNobel) [reply mt2.tex:297]: And the rest of the doc

**Review by PTNobel — CHANGES_REQUESTED** [review]
This is far from a complete review, but this was an initial pass. If @GeoHutch wants me to do the fixes on the syntax mistakes I'm happy to when we merge it, just wanted to note the issues so they don't get lost.

**Comment by GeoHutch** [comment]
I'm going to make some small changes (including the ones Parth brings up) and merge this, so we can have a completed rough draft to continue working from as of tonight.

**Pull request merged**